### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by dathanb** [sirwart/secrets#12]
I just tested `secrets` on my repo, and it did a really good job! It discovered a couple of (test mode) unencrypted private keys, and flagged a bunch of public key fingerprints that definitely look like secrets. Kudos on the tool!

In case you're looking at tweaking the heuristics, I wanted to call out this line it flagged, which is clearly a false positive:
```
if ((filterMode == FilterMode.OFFLINE_STORAGE && key.isPersistent()) || (filterMode == FilterMode.API_RETURN_VALUE && key.isReturnable())
```

Thanks!

**Comment by sirwart** [comment]
Thanks for reporting @dathanb! I'm definitely interested in reducing false positives, so I will add a fix for this when I get a chance.

**Comment by sirwart** [comment]
Someone else reported a false positive, and the fix actually worked for both issues (https://github.com/sirwart/secrets/commit/e3f7bf005164f35cc08a3a4683473d09928c6617). The fix will go out in 0.1.3 which should be released tomorrow
